src/components/Question.tsx

```diff
@@ -12,25 +12,31 @@ import { Question as QuestionType } from '../types';
 function Question({ question }: { question?: QuestionType }) {
   return (
     <>
-        <QuestionNumberContainer>
-          QUESTION
-          <QuestionNumber>0013</QuestionNumber>
-        </QuestionNumberContainer>
-      <QuestionTitle>{question?.title}</QuestionTitle>
-      <QuestionDescription>{question?.content}</QuestionDescription>
+      <QuestionNumberContainer>
+        QUESTION
+        <QuestionNumber>0013</QuestionNumber>
+      </QuestionNumberContainer>
+
+      <QuestionTitle>Capital of Spain</QuestionTitle>
+      <QuestionDescription>Marry is traveling to Spain for her vacation. She is wondering which of the following cities is the capital of Spain?</QuestionDescription>
```

Comment:
You could return the dummy question for now as a fixed return value inside `DefaultApiClient` (maybe with a To Do comment) and leave it as the "proper code" here... But this approach is also ok ;) Just as an idea

Reply:
Yes, this is a good idea. I now creates a `DummyApiClient` for this.